docs/introduction/start-building-your-application.md

```diff
@@ -113,6 +113,24 @@ switch ($domainId) {
     case 3: $defaultCurrency = $this->getReference(CurrencyDataFixture::CURRENCY_CZK); break;
     ...
 ```
+
+## Fine-tune your configuration
+If all developers working on your project use the same version of PHP (eg. because all use SSFW via Docker), you can use higher versions of the libraries and tools installed via Composer.
```

Comment:
```suggestion
If all developers working on your project use the same version of PHP (e.g. because all use SSFW via Docker), you can use higher versions of the libraries and tools installed via Composer.
```